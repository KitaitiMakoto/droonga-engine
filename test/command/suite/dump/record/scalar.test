#@require-catalog-version 2
#@disable-logging
{
  "type": "add",
  "dataset": "Default",
  "body": {
    "table": "Users",
    "key": "Adam"
  }
}
#@enable-logging
#@subscribe-until 1s
{
  "type": "dump",
  "dataset": "Default"
}
