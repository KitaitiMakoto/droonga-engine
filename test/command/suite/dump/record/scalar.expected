{
  "inReplyTo": "request-id",
  "dataset": "Default",
  "date": "0000-00-00T00:00:00.000000Z",
  "type": "dump.start"
}
{
  "inReplyTo": "request-id",
  "dataset": "Default",
  "date": "0000-00-00T00:00:00.000000Z",
  "type": "dump.start"
}
{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "dump.result",
  "body": {
    "started": true
  }
}
{
  "inReplyTo": "request-id",
  "dataset": "Default",
  "date": "0000-00-00T00:00:00.000000Z",
  "body": {
    "nMessages": 1
  },
  "type": "dump.forecast"
}
{
  "inReplyTo": "request-id",
  "dataset": "Default",
  "date": "0000-00-00T00:00:00.000000Z",
  "body": {
    "nMessages": 2
  },
  "type": "dump.forecast"
}
{
  "inReplyTo": "request-id",
  "dataset": "Default",
  "date": "0000-00-00T00:00:00.000000Z",
  "body": {
    "type": "Hash",
    "name": "Users",
    "keyType": "ShortText"
  },
  "type": "dump.table"
}
{
  "inReplyTo": "request-id",
  "dataset": "Default",
  "date": "0000-00-00T00:00:00.000000Z",
  "body": {
    "type": "Hash",
    "name": "Users",
    "keyType": "ShortText"
  },
  "type": "dump.table"
}
{
  "inReplyTo": "request-id",
  "dataset": "Default",
  "date": "0000-00-00T00:00:00.000000Z",
  "body": {
    "table": "Users",
    "key": "Adam",
    "values": {
    }
  },
  "type": "dump.record"
}
{
  "inReplyTo": "request-id",
  "dataset": "Default",
  "date": "0000-00-00T00:00:00.000000Z",
  "type": "dump.end"
}
{
  "inReplyTo": "request-id",
  "dataset": "Default",
  "date": "0000-00-00T00:00:00.000000Z",
  "type": "dump.end"
}
